### ISSUE TITLE
Use Flutter SDK as project configuration 

### ISSUE THREAD

**Issue by buijs-dev** [buijs-dev/klutter#23]
TLDR: Store the Flutter SDK version in the Klutter project and use a default location for Flutter SDK distributions so that the Klutter Gradle Plugin knows where to find it.

Running flutter commands from Gradle using global flutter can result in compatibility issues. A Klutter project should be build and run by all users with the same Flutter SDK. Bumping the Flutter SDK version should be a deliberate action.

Using an explicit path to Flutter SDK should also fix the windows issues where global flutter is not recognized.

Required changes:
- Keep track of compatible Flutter SDK versions (major + minor + patch versions, platform Macos/Windows and architecture.
- Write utility methods which can download the correct Flutter SDK to a predetermined folder (user.home/KlutterProjects/.cache). 
- Store Flutter SDK version in the build.gradle.kts.
- Let all Gradle tasks use the cached Flutter SDK distribution.
- Add Flutter download option to kommand cli tool.

Acceptance Criteria:
- Generating a project with kommand or Jetbrains IDE should have an option to specify a Flutter SDK version.
- When a Flutter SDK version is already present in the cache then it is not redownloaded.
- Cache can be emptied using kommand and/or Jetbrains IDE.
- All flutter commands are executed using the correct Flutter SDK from cache folder as specified in the build.gradle.kts.
